BUG: fixed two minor typos

cutrightjm opened the pull request:
Fixed two minor typos

Signed-off-by: Jacob M Cutright < cutrightjm@stl.sh >

cutrightjm committed:
Fixed two minor typos

pcmoore commented:
Thanks @cutrightjm for the fix, but would you mind signing your work as described in [CONTRIBUTING.md](https://github.com/SELinuxProject/selinux-notebook/blob/main/CONTRIBUTING.md)?

cutrightjm committed:
Update global_config_files.md

Updated REQUIREUSERS -> REQUIRESEUSERS. There are no references to REQUIREUSERS in the source.

pcmoore commented:
Hi @cutrightjm, I apologize that it has taken so long to revisit this PR, but I still don't see your sign-off in the patches?  Here is an example taken from commit b96e479d5a97de782e7254ad6c3f53fbd2969b3d:
```
commit b96e479d5a97de782e7254ad6c3f53fbd2969b3d
Author: Richard Haines <richard_c_haines@btinternet.com>
Date:   Mon Dec 13 10:16:19 2021 +0000

    selinux-notebook: Make file links consistent
    
    Before building EPUB book the file links to the notebook-examples
    directory area needs to be consistent. This will allow sed to fix the path
    during the build (as pandoc insists on defaulting to file:///EPUB/text).
    
    Signed-off-by: Richard Haines <richard_c_haines@btinternet.com>
```

pcmoore commented:
Thanks @cutrightjm, these patches were merged at a844d81c3a3a818ee1149db6a1292b4c08929213.
